src/components/InventoryTabs/ConventionalSystems/ConventionalSystemsTab.js

```diff
@@ -180,7 +180,7 @@ const ConventionalSystemsTab = ({
 
       if (isKesselEnabled) {
         return selectedHosts.every(
-          ({ groups }) => groups[0].ungrouped !== true,
+          ({ groups }) => groups[0].ungrouped === true,
```

Comment:
I suppose it should be given, but should we check to make sure that the host is only in the ungrouped group?
```suggestion
          ({ groups }) => groups.length === 1 && groups[0].ungrouped === true,
```

Reply:
not a bad idea. I've changed it to `groups[0]?.ungrouped === true`. This will return false if `groups[0]` doesn't exist.

Reply:
that covers the case for lenght < 0 and avoids errors.
Case where we have 2 groups should not happen I assume? :)